### PR TITLE
feat: google anaytics 4 and matomo for OFF

### PR DESCRIPTION
**Pull request by alexgarel**
### Checklist

- use GA4 with as much privacy as possible
- setup Matomo to eventually replace GA4

### CODE REVIEW

File: lib/ProductOpener/Config_off.pm [hunk 1]
```diff
@@ -357,14 +357,35 @@ $page_size = 24;
 
 $google_analytics = <<HTML
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-31851927-1"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-HQX9SYHB2P&aip=1"></script>
```
Comment (stephanegigandet) [inline lib/ProductOpener/Config_off.pm:360]: It's not possible to keep the same id so that we have continuous stats?

Comment (alexgarel) [reply lib/ProductOpener/Config_off.pm:360]: No, this is a requirement of GA4
Moreover GA4 drastically decrease retention period to 2 monthes (but we could set 4 monthes) (@teolemon you are aware of it right ?)

Comment (stephanegigandet) [reply lib/ProductOpener/Config_off.pm:360]: ah ok, well if it retains data for only a few months anyway, then this point becomes moot.

but that means that we can't even have very aggregated metrics across years? e.g. getting a graph with monthly users and page views over a period of several years?

Comment (stephanegigandet) [reply lib/ProductOpener/Config_off.pm:360]: It seems it's only for exploration: if you want to slice and dice data accross users: https://support.google.com/analytics/answer/7667196?hl=fr

But the summary metrics are kept.

Comment (stephanegigandet) [reply lib/ProductOpener/Config_off.pm:360]: ok, anyway there's no choice, so let's go for it

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**